metrics/test/test_file_length.py

```diff
@@ -29,14 +29,14 @@ def test_file_length_given_java_file_returns_1(self):
         result = file_length.run()
         self.assertEqual(result.get_frequency(), 1)
 
-    def test_file_length_given_csharp_file_returns_1(self):
+    def test_file_length_given_csharp_file_returns_2(self):
         new_source_repository = SourceRepository(
             Path("./metrics/test/data/test_data_file_length"), "c_sharp"
         )
         file_length = FileLength()
         file_length._source_repository = new_source_repository
         result = file_length.run()
-        self.assertEqual(result.get_frequency(), 1)
+        self.assertEqual(result.get_frequency(), 3)
```

Comment:
Number doesn't match method name

Reply:
The number is correct the method name should be updated

Reply:
fixed